.github/workflows/PRLabeling.yml

```
@@ -15,22 +16,148 @@ jobs:
+            -H "Accept: application/vnd.github+json" \
+            https://api.github.com/repos/${{ github.repository }}/issues/${{ github.event.pull_request.number }}/labels \
+            -d '["${{ env.LABEL }}"]'
+
+  label_existing_prs:
+    runs-on: ubuntu-latest
+    needs: label_pr
+    if: github.event_name == 'workflow_dispatch'  # Only trigger when manually run
+
+    steps:
+      - name: Fetch all open PRs
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:60:66: Double quote to prevent globbing and word splitting

(shellcheck)